Fix typos and grammar in README.md (#43)
willmcgugan/textual: README.md

await self.view.dock(Placeholder(), Placeholder(), edge="top")
```

> You will notice that this time we are docking _two_ Placeholder objects on to the `"top"` edge. We haven't set an explicit size this time so Textual will divide the remaining size amongst the two new widgets.

Reviewer: ```suggestion
You will notice that this time we are docking _two_ Placeholder objects onto the `"top"` edge. We haven't set an explicit size this time so Textual will divide the remaining size amongst the two new widgets.
```

i think slightly improved with `onto`

I don't see your change as strictly necessary here, though.
